feature/preview/src/main/res/values/strings.xml

```diff
@@ -23,4 +23,8 @@
 
     <string name="toast_image_capture_success">Image Capture Success</string>
     <string name="toast_capture_failure">Image Capture Failure</string>
+    <string name="stabilization_icon_description_preview_and_video">Preview and video are stabilized</string>
+    <string name="stabilization_icon_description_preview_only">Only Preview is Stabilized</string>
```

Comment:
This shouldn't be a description. As discussed in other comments, preview can't be stabilized by itself.